### ISSUE TITLE
"fg ema" lacks "%" mark in Process expansion example

### ISSUE THREAD

**Issue by ymmtmdk** [fish-shell/fish-site#40]
Hi,
In Process expansion part of index.html
https://github.com/fish-shell/fish-site/blob/master/site/docs/current/index.html#L636

It is written with "fg ema will search for ..." 
Should this be "fg %ema will search for ..." ?

**Comment by krader1961** [comment]
Yes. What's interesting is the percent symbol is in the *doc_src/index.hdr.in* source file. And the following line that references `%3` is correctly rendered. So probably something in the doxygen lexicon filtering script is broken. Also, whomever fixes this should augment the *doc_src/fg.txt* document to link to this portion of the docs and at least mention that things other than PIDs are allowed.

**Comment by faho** [comment]
CC @MarkGriffiths.

**Comment by MarkGriffiths** [comment]
It's not down to the lexicon_filter as it only highlights syntax on content within `\fish` … `\endfish` blocks.

It seems to be a Doxygen 'special case'. From Doxy's docs:

```
\% : This command writes the % character to the output. This character has to be
escaped in some cases, because it is used to prevent auto-linking to word
that is also a documented class or struct.
```

What's curious is that the specific combination of `%em` is being interpreted as a special command when in backticks and escaping it normally prints the backslash.

Easy fix: make it <code>`fg %%ema`</code> in `index.hdr.in`

**Comment by zanchey** [comment]
Fixed with the merge of fish-shell/fish-shell#3247 - thanks for reporting the issue @ymmtmdk!

**Comment by ymmtmdk** [comment]
Your welcome. Thank you for your work!
